src/components/TeamMemberList.js

```diff
@@ -1,31 +1,41 @@
-import { useState, useEffect } from "react";
+import { useState, useEffect, useCallback } from "react";
 import TeamMember from "./TeamMember";
 import axios from "axios";
+import apiUrl from "../api/paths";
 
 const TeamMemberList = (props) => {
   const [users, setUsers] = useState([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
 
-  useEffect(() => {
-    const fetchUsers = async () => {
-      const response = await axios.get(process.env.REACT_APP_ENDPOINT_URL);
+  const fetchUsers = useCallback(async () => {
+    try {
+      setIsLoading(true);
+      const response = await axios.get(`${process.env.REACT_APP_ENDPOINT_URL}${apiUrl.USERS}`);
       setUsers(response.data);
-    };
-    fetchUsers();
+    } catch (error) {
+      setIsError(true);
+    }
+    setIsLoading(false);
```

Comment:
It will work, but I usually add `setIsLoading` inside `finally` block, which will be executed after async (resolved or rejected). For me it's connects the whole method in one almost sync section :)

Reply:
Ok, that's what the finally is used for! :) Good example, i've changed the code.